docs/apache-airflow-providers-github/commits.rst

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
This one can be removed or left empty - it should be updated after merge - during release, otherwise it will have wrong commit id (but maybe it is needed to stop test from failing in which case it can remain as it is).

Reply:
sure, keeping it as is for now.
we can update this upon release.